.github/sync-files.yaml

```diff
@@ -89,3 +89,4 @@
   files:
     - source: .github/workflows/deploy-docs.yaml
     - source: .github/workflows/delete-closed-pr-docs.yaml
+    - source: mkdocs.yaml
```

Comment:
@satoshi-ota You need to add some `pre-commands` to avoid manual fixes like https://github.com/autowarefoundation/autoware.universe/pull/3619/commits/a105a66ebb633f303f2cd8cf58b19d260b1bc488.
https://github.com/autowarefoundation/open-ad-kit-docs/blob/0c1c70625fb89d144a5e2d5df52d0cdccd6e18c4/.github/sync-files.yaml#L37-L41

Reply:
Ah, I see. I'll try it :+1: